.gitignore

```diff
@@ -11,6 +11,7 @@ capybara-*.html
 *.orig
 rerun.txt
 pickle-email-*.html
+/test
```

Comment:
Much easier to just use the [bootstrap .gitignore](https://github.com/Wolox/rails-bootstrap/blob/master/.gitignore) instead of adding stuff here all the time: